packages/format-library/src/text-color/index.js

```diff
@@ -15,13 +15,44 @@ import { removeFormat } from '@wordpress/rich-text';
 /**
  * Internal dependencies
  */
-import { default as InlineColorUI, getActiveColor } from './inline';
+import { default as InlineColorUI, getActiveColors } from './inline';
 
 const name = 'core/text-color';
-const title = __( 'Text color' );
+const title = __( 'Color' );
```

Comment:
Can we try adjusting the tool name from "Color" to "Highlight"? I think color alone is too confusing, especially with block level color in the sidebar.